### DISCUSSION TITLE
Json tags cannot be filtered by content (is one of) or searched for

### DISCUSSION THREAD

**Discussion by nordcart** [directus/directus#12303] [Archive]
Title tells the issue

![image](https://user-images.githubusercontent.com/12813484/159463774-6405c258-82ff-4cdc-beb6-033cc1f02bd6.png)

![image](https://user-images.githubusercontent.com/12813484/159463869-7946a9e9-027f-4d3b-a503-9e509bb233d4.png)

**Comment by rijkvanzanten** [comment]
That's correct. JSON can't yet be searched through. See https://github.com/directus/directus/discussions/7277 as well
